### PR TITLE
[GUVNOR-3016] New Kie Maven Takari Plugin

### PR DESCRIPTION
**Pull request by desmax74**
New Kie maven takari plugin.
PR related kiegroup/droolsjbpm-integration#1030 and kiegroup/kie-wb-common#941

### CODE REVIEW

**Comment by porcelli** [comment]
@ederign github automatically squash the commits :) - learned this today

**Comment by ederign** [comment]
@porcelli I know, but should we start doing this?

**Comment by desmax74** [comment]
PR related https://github.com/kiegroup/droolsjbpm-integration/pull/1030 and https://github.com/kiegroup/kie-wb-common/pull/941

File: pom.xml [hunk 1]
```diff
@@ -2325,7 +2327,20 @@
...
         <version>${version.org.jboss.narayana.tomcat}</version>
       </dependency>
-    </dependencies>
+    
+    <!-- Incremental compiler-->
+      <dependency>
+        <groupId>io.takari.maven.plugins</groupId>
+        <artifactId>takari-lifecycle-plugin</artifactId>
+        <version>${version.io.takari.maven.plugins}</version>
+      </dependency>
```
Comment (psiroky) [inline pom.xml]: I am not completely sure, but I think this should go into `pluginMgmt`.

Comment (desmax74) [inline pom.xml]: I have the same feeling let me try if works moving in the pluginMgmt

Comment (desmax74) [inline pom.xml]: Moved into pluginManagement

**Pull request merged**